routes/error-tracker.js

```diff
@@ -129,7 +129,7 @@ async function handler(req, res) {
     event = await buildEvent(req, reportingParams, logTarget);
   } catch (unminifyError) {
     console.error(unminifyError);
-    return res.sendStatus(statusCodes.ACCEPTED);
+    return res.sendStatus(statusCodes.INTERNAL_SERVER_ERROR);
```

Comment:
Nit, actually, I think a 422 would be semantically better here. https://www.iana.org/assignments/http-status-codes/http-status-codes.xhtml

Reply:
For a sec I thought you were joking w/ 418 :tea:

Reply:
Done.